gotool/gotool.go

```diff
@@ -7,6 +7,19 @@ import (
 	"github.com/bitrise-io/go-utils/command"
 )
 
+// CommandExecutor ...
+type CommandExecutor struct{}
+
+// ExecuteCommand ...
+func (c CommandExecutor) ExecuteCommand(stringCommand string, args ...string) (string, error) {
```

Comment:
Can we integrate this into `"github.com/bitrise-io/go-utils/command"` somehow? I'm afraid that these additional layers will get added to more repos and we forget about them and that leaves a lot of inconsistent duplicate code.
If not, can we check if we really need this new type?

Reply:
Let's not block this PR for now, but in the future we should come up with a common solution for this